native/chat/chat-thread-background.react.js

```diff
@@ -15,6 +20,35 @@ type BackgroundChatThreadListProps = {
 export default function BackgroundChatThreadList(
   props: BackgroundChatThreadListProps,
 ) {
+  const chatThreadListData = useSelector(state => chatListData(state));
+
+  const findUnreadBackgroundThreadsNumber = (
+    chatThreadsListData: Array<ChatThreadItem>,
+  ): number => {
+    return chatThreadsListData
+      .map(info => ({
+        home: info.threadInfo.currentUser.subscription.home,
+        unread: info.threadInfo.currentUser.unread,
+      }))
+      .filter(item => !item.home && item.unread).length;
+  };
+
+  const prevUnreadNumber = React.useRef(0);
+  const unreadBackgroundThreadsNumber = React.useMemo(() => {
+    return findUnreadBackgroundThreadsNumber(chatThreadListData);
+  }, [chatThreadListData]);
+
+  React.useEffect(() => {
+    if (unreadBackgroundThreadsNumber == prevUnreadNumber.current) return;
```

Comment:
Some nits:

1. Let's always default to using the `===` comparison
2. Let's always include curly braces for blocks, and put the contents on a new line:

```js
if (unreadBackgroundThreadsNumber === prevUnreadNumber.current) {
  return;
}
```